packages/patternfly-3/patternfly-react-wooden-tree/src/components/TreeView/TreeView.stories.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info/dist/index';
+import { defaultTemplate } from 'storybook/decorators/storyTemplates';
+import { storybookPackageName } from 'storybook/constants/siteConstants';
+import { name } from '../../../package.json';
+import { TreeView } from './index';
+
+const stories = storiesOf(`${storybookPackageName(name)}/Tree View`, module);
+
+stories.addDecorator(
+  defaultTemplate({
+    title: 'Catalog Tile',
+    description:
+      'This is a Tree View component to display trees and hierarchical data using React Wooden Tree.' +
+      'The tree uses the https://www.npmjs.com/package/react-wooden-tree component.'
+  })
+);
+
+stories.add(
+  'Tree View',
+  withInfo({
+    source: false,
+    propTables: [TreeView]
+  })(() => (
+    <div style={{ display: 'flex' }}>
+      <TreeView />
```

Comment:
Once we've moved or imported the test data in this file, I think we want this to be

```js
<TreeView data={data} />
```